.github/workflows/build.yml

```diff
@@ -64,8 +64,14 @@ jobs:
             web/server/h2o/libh2o/
           files_ignore: |
             netdata.spec.in
-            **.md
-          negation_patterns_first: true
+            **/*.md
+      - name: List all changed files in pattern
+        env:
+          ALL_CHANGED_FILES: ${{ steps.check-files.outputs.all_changed_files }}
```

Comment:
I think we need to use `.any_modified` because later we use it to decide whether to skip the checks.